ui5/browser/controller/Browser.controller.js

```diff
@@ -714,6 +714,72 @@ sap.ui.define(['sap/ui/core/mvc/Controller',
       /* =============== TabContainer =============== */
       /* ============================================ */
 
+      /* ======================================== */
+      /* =============== Terminal =============== */
+      /* ======================================== */
+
+      onTerminalSubmit: function(oEvent) {
+         let command = oEvent.getSource().getValue();
+         let url = '/ProcessLine/cmd.json?arg1="' + command + '"';
+         console.log(command);
+         this.websocket.Send("CMD:" + command);
+         oEvent.getSource().setValue("");
+         this.requestRootHist();
+         this.requestLogs();
+      },
+
+      requestRootHist: function() {
+         return this.websocket.Send("ROOTHIST:");
+      },
+
+      updateRootHist: function (hist) {
+         let pos = hist.lastIndexOf(',');
+         hist = hist.substring(0,pos) + "" + hist.substring(pos+1);
+         hist = hist.split(",");
+         let json = {hist:[]};
+
+         for(let i=0; i<hist.length; i++) {
+            json.hist.push({name: hist[i] });
+
+         }
+         this.getView().byId("terminal-input").setModel(new JSONModel(json));
+      },
+
+      requestLogs: function() {
+         return this.websocket.Send("LOGS:");
+      },
+
+      updateLogs: function(logs) {
+         this.getView().byId("output_log").setValue(logs);
+      },
+
+      updateLog: function() {
```

Comment:
This function is not used - why it here?

Reply:
Oh my bad it is some remains of past functions, i remove the asap

Reply:
Done